packages/dd-trace/src/config.js

```diff
@@ -295,6 +295,10 @@ class Config {
     const DD_TRACE_SPAN_ATTRIBUTE_SCHEMA = validateNamingVersion(
       process.env.DD_TRACE_SPAN_ATTRIBUTE_SCHEMA
     )
+    const DD_TRACE_REMOVE_INTEGRATION_SERVICE_NAMES_ENABLED = coalesce(
```

Comment:
We should probably be a bit more explicit on requiring a human-wise truthy value - that's perhaps as simple as `coalesce(isTrue(process.env.DD_TRACE...), false)`

Reply:
did this